Yoast/wpseo-news
# Dont allow taxonomies without show ui to be excluded

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where prominent words would be shown in the list of terms to be excluded from the news sitemap.

## Relevant technical choices:

* Create a class `WPSEO_Exclude_Taxonomies` with a static helper function to retrieve all Taxonomies of which the terms are eligible for exclusion. 
* After consulting with @jip and @jdevalk we decided to exclude taxonomies from the list of with the `show_ui` parameter is false.

## Test instructions

This PR can be tested by following these steps:

* Make sure you use Yoast SEO Premium.
* Before you check out this branch check whether you have prominent words in the News SEO admin page, under `Terms to exclude for Posts`.
* If not create a few posts with similar content, and check again.
* Check out this branch, make sure the prominent words no longer apprear.

Fixes #439
